src/coffee/connect/rest.coffee

```diff
@@ -169,8 +169,6 @@ class Rest
 
   # Public: Fetch all results of a Sphere resource query endpoint in batches of pages using a recursive function.
   #
-  # Note that traversing with pagination has been optimized. It now fetches pages sorted by `id` and iterates
-  # using a query with the `id` greater then the last one of the previous page.
```

Comment:
How about this:
```coffee
  # Note that in order to correctly traverse with pagination you need to provide a sorting.
  # If you use the all() method and don't provide any sorting, it will use sorting by id automatically.
  # With proper sorting PAGED fetches pages and iterates
  # using a query with the `id` greater then the last one of the previous page.
```